### ISSUE TITLE
Replace xml2json

### ISSUE THREAD

**Issue by alebelcor** [alebelcor/gigs-adapter-stackoverflow-jobs#1]
Replace `xml2json` with something that doesn't require native compilation.

Like `xml2js` or some other package. It can even be a callback-style helper, it can be promisified using `pify`.

Once this is done, make sure the added stuff (for compilation) to `.travis.yml` is removed.

**Comment by alebelcor** [comment]
Fixed by 82d7da520bc5c8a7a19541cedca7b39c96c79830

**Comment by alebelcor** [comment]
I'm still not happy with the output.

Replace `xml2js` with `feedme`.

``` js
const Feedme = require('feedme');
const parser = new Feedme(true);

parser.write(fs.readFileSync('./rss.xml'));

const json = parser.done();
```

**Comment by alebelcor** [comment]
This was fixed right before `1.0.0` in 5dbc8add81d5ea6304be69622d739cd018b13393
